### PR TITLE
add cases OCP-43146

### PR DESCRIPTION
**Pull request by zhaozhanqi**
add cases for this bug https://bugzilla.redhat.com/show_bug.cgi?id=1995871

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
logs: http://pastebin.test.redhat.com/990286

**Comment by zhaozhanqi** [comment]
could you help take a look?  @anuragthehatter

**Comment by zhaozhanqi** [comment]
for upgrade cases:

prepare: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Runner/157964/
check :  https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Runner/157973/console

**Comment by zhaozhanqi** [comment]
@openshift/team-sdn-qe PTAL, thanks

**Comment by zhaozhanqi** [comment]
Added nodeport service
upgrade prepare: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Runner/159787
upgrae check: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Runner/159788/console

File: features/networking/sdn.feature [hunk 1]
```diff
@@ -454,3 +454,18 @@ Feature: SDN related networking scenarios
...
+    Given the env is using "OpenShiftSDN" networkType
+    Given I select a random node's host
+    And I run commands on the host:
+      | iptables -t raw -S |
+    Then the step should succeed
+    And the output should contain:
+      | -N OPENSHIFT-NOTRACK |
+      | -A PREROUTING -m comment --comment "disable conntrack for vxlan" -j OPENSHIFT-NOTRACK |
+      | -A OUTPUT -m comment --comment "disable conntrack for vxlan" -j OPENSHIFT-NOTRACK |
+      | -A OPENSHIFT-NOTRACK -p udp -m udp --dport 4789 -j NOTRACK |
```
Comment (liangxia) [inline features/networking/sdn.feature:471]: Tab align issue.

Comment (zhaozhanqi) [reply features/networking/sdn.feature]: fixed

**Pull request merged**